changelogs/unreleased/changelogs/unreleased/8304-update-installation-instructions-native-docker-container.yml

```diff
@@ -0,0 +1,9 @@
+description: Update installation instructions for new native docker container
+issue-nr: 8304
+change-type: patch
+destination-branches: [master]
```

Comment:
You are missing the `issue-repo` here, because the repo of the PR is not the same as the repo of the issue.